components/dbutil/src/main/java/com/wso2telco/core/dbutils/DbService.java

```diff
@@ -15,18 +15,18 @@
  ******************************************************************************/
 package com.wso2telco.core.dbutils;
 
-import java.sql.*;
-import java.util.*;
-import java.util.Date;
-
-import javax.persistence.PersistenceException;
-
 import com.wso2telco.core.dbutils.dao.SpendLimitDAO;
 import com.wso2telco.core.dbutils.model.FederatedIdpMappingDTO;
-
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
+import javax.persistence.PersistenceException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.*;
```

Comment:
Are these related to change for - INTGW-797 | Sonar issues fixing. Added try-with-resources

Reply:
Yes